recipe/meta.yaml

```
@@ -42,7 +43,6 @@ requirements:
     - packaging
   run:
     - python
-    - {{ pin_compatible('numpy') }}
     - scipy
```

```suggestion
    - numpy >=1.22.4
    - scipy >=1.8.1
```

you might want to update your other pins as well